src/Controller/ContactFormController.php

```diff
@@ -50,22 +53,15 @@ final class ContactFormController
     private $contactFormRepository;
     /** @var TokenStorageInterface */
     private $token;
-    /** @var bool */
-    private $sendManager;
-    /** @var bool */
-    private $sendCustomer;
-    /** @var bool */
-    private $nameRequired;
-    /** @var bool */
-    private $phoneRequired;
     /** @var string */
     private $recaptchaPublic;
     /** @var string */
     private $recaptchaSecret;
 
     public function __construct(
-        TranslatorInterface $translator,
+        ContactFormSettingsProvider $contactFormSettings,
```

Comment:
There use interface  `ContactFormSettingsProviderInterface`

Did you completely delete the validation?

Reply:
DONE